pool/chainstate_test.go

```diff
@@ -3,6 +3,7 @@ package pool
 import (
 	"context"
 	"encoding/hex"
+	"errors"
```

Comment:
wondering what triggered this addition considering the change made does not use anything from the `errors` package.

Reply:
Using `errors.Is` to confirm correct error types. Maybe the import was included in the wrong commit because I commited things in the wrong order.

Reply:
yeah must have slipped through when you were splitting the work done into task focused commits.